acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ThreadBesuNodeRunner.java

```diff
@@ -237,35 +178,20 @@ public void startNode(final BesuNode node) {
             .transactionPoolValidatorService(transactionPoolValidatorServiceImpl)
             .build();
 
+    final BesuControllerBuilder builder = component.besuControllerBuilder();
+    builder.isRevertReasonEnabled(node.isRevertReasonEnabled());
+    builder.networkConfiguration(node.getNetworkingConfiguration());
```

Comment:
nit: this made me double take - is there a Dagger-y reason not to chain these or just a style thing?

Reply:
No, no reason that would be dagger specific.